packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -391,6 +391,9 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
             _logger.finer('***batchId:$batchId key: ${commitEntry.atKey}');
             await syncUtil.updateCommitEntry(
                 commitEntry, commitId, _atClient.getCurrentAtSign()!);
+            // update the last received server commit id
+            await _atClient.put(
```

Comment:
sometimes commitId can be -1 when there is no response from server or server returns data:-1. 
I guess we  should not update the lastReceivedServerCommitId in this case

Reply:
Thanks, Murali. Added IF condition and pushed the changes.